src/components/vuestic-components/va-icon/VaIcon.vue

```diff
@@ -95,5 +99,19 @@ export default {
 .va-icon {
   vertical-align: middle;
   user-select: none;
+
+  &.spin {
+    animation: rotate 1000ms infinite;
+  }
+}
+
+@keyframes rotate {
```

Comment:
Please use something similar to [this solution](https://github.com/epicmaxco/epic-spinners/blob/master/src/services/utils.js).
We ideally do not want to introduce any global css.